website/docs/r/emrcontainers_virtual_cluster.markdown

```diff
@@ -57,8 +57,10 @@ In addition to all arguments above, the following attributes are exported:
 
 ## Import
 
-EKS Clusters can be imported using the `name`, e.g.
+EKS Clusters can be imported using the `id`, e.g.
 
 ```
-$ terraform import aws_emrcontainers_virtual_cluster.example
+$ aws emr-containers list-virtual-clusters | jq -r '.virtualClusters[].id'
+> a1b2c3d4e5f6g7h8i9j10k11l
```

Comment:
I think we should omit these two lines to keep this example in line with the existing import examples (for other resources)

Reply:
I see your point and adjusted it accordingly